### PR TITLE
Mme calculator frontend updates

### PR DESCRIPTION
**Pull request by achen2401**
- include new MMECalculator CQL library to calculate MME values for MedicationRequests
- feed MED graph with real data generated from CQL
- clean MedicationRequest code in  R4 CQL files related to the status field
- fix Education Material section to display both patient and provider education materials in different sections
- add concrete date to the alert tex in the overview panel where possible

[Launch URL](https://launch.smarthealthit.org/launcher?launch_uri=https%3A%2F%2Fdeploy-preview-61--distracted-jepsen-bc9c39.netlify.com%2Flaunch.html&patient=5c41cecf-cf81-434f-9da7-e24e5a99dbc2&fhir_ver=4)

### CODE REVIEW

File: src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql [hunk 1]
```diff
@@ -977,6 +997,55 @@ define ReportMostRecentMME:
...
+  sort by End desc
+
+define ReportMMEByDates:
+(ReportMME) R
+let Filter_List: R where R.End is not null and R.MMEValue is not null
+return {
+  End: Filter_List.End,
+  MMEValue: Sum(ReportMME M where M.End = Filter_List.End return M.MMEValue)
+}
+sort by End
```
Comment (mldorr) [inline src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql:1044]: these should be indented.

Comment (achen2401) [reply src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql]: thank you, fixed

File: src/components/Landing.js [hunk 2]
```diff
@@ -211,8 +215,25 @@ export default class Landing extends Component {
...
+    //process graph data
+    if (overviewSection.graphConfig && overviewSection.graphConfig.summaryDataSource) {
+      //get the data from summary data
+      let sections = overviewSection.graphConfig.summaryDataSource;
+      let graph_data = [];
+     
+      sections.forEach(item => {
+        if (summary[item.section_key] && summary[item.section_key][item.subSection_key]) {
+          //console.log("section data? ", summary[item.section_key][item.subSection_key])
+          graph_data = [...graph_data, ...summary[item.section_key][item.subSection_key]];
```
Comment (ivan-c) [inline src/components/Landing.js:230]: Cool, wasn't aware of [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters)!

**Review by ivan-c — APPROVED** [review]
Looks good! A little over my head but didn't see any blockers and tested the launch URL successfully

**Pull request merged**